internal/testutil/tempdir.go

```diff
@@ -83,3 +85,17 @@ func (td *TempDir) WriteYAML(file string, obj interface{}) *TempDir {
 	}
 	return td.Write(file, content)
 }
+
+func (td *TempDir) InitEmptyGitRepo(path, url string) {
```

Comment:
if you search for `gitutil.Exec(... ,"init")` you're likely to find that we use this code in so many other places. would be good to consolidate.

I'm +1 for this method. But if we actually add a file + commit it, then the index can actually be `git clone`d. (But the method name needs to change obviously.)

Reply:
I see you updated quite a few tests up there, maybe that's actually all of it.